addons/purchase/models/purchase_order_line.py

```diff
@@ -678,3 +678,7 @@ def _get_select_sellers_params(self):
         return {
             "order_id": self.order_id,
         }
+
+    def _get_cleaned_tax_names(self):
+        """ Return a comma-separated string of the taxes names, without HTML tags. """
+        return ', '.join(html2plaintext(tax.description or tax.name) for tax in self.taxes_id)
```

Comment:
since `tax.name`  is a Char it seems it doesn't need `html2plaintext` also testing `html2plaintext` it returns an empty string for `<p></p>` or `<p><br/></p>`, so doing:

```suggestion
        return ', '.join(html2plaintext(tax.description) or tax.name for tax in self.taxes_id)
```
should have the advantage on fallbacking on name if description is an empty html tag.